src/toil/provisioners/clusterScaler.py

```diff
@@ -477,10 +478,10 @@ def getEstimatedNodeCounts(self, queuedJobShapes, currentNodeCounts):
                                           goalTime=self.targetTime)
         estimatedNodeCounts = {}
         for nodeShape in self.nodeShapes:
-            nodeType = self.nodeShapeToType[nodeShape]
+            instance_type = self.nodeShapeToType[nodeShape]
 
             logger.debug("Nodes of type %s to run queued jobs = "
```

Comment:
I think the way it is makes more sense. I guess the other way distinguishes better between the node type and instance type concepts. But if we made it say "Instances of type blah" that would be weird, since Toil talks about nodes throughout. And saying that the nodes have "instance_type blah" is kind of ugly.